on_the_web.module

```diff
@@ -255,39 +262,44 @@ function on_the_web_get_services() {
   $services = array(
     'twitter' => array(
       'name' => 'Twitter',
-      'fa-icon' => 'fa-twitter-square',
+      'fa-icon' => 'fa-twitter',
+      'fa-icon-square' => 'fa-twitter-square',
     ),
     'facebook' => array(
       'name' => 'Facebook',
-      'fa-icon' => 'fa-facebook-square',
+      'fa-icon' => 'fa-facebook',
+      'fa-icon-square' => 'fa-facebook-square',
     ),
     'pinterest' => array(
       'name' => 'Pinterest',
-      'fa-icon' => 'fa-pinterest-square',
+      'fa-icon' => 'fa-pinterest',
+      'fa-icon-square' => 'fa-pinterest-square',
     ),
     'instagram' => array(
       'name' => 'Instagram',
       'fa-icon' => 'fa-instagram',
     ),
     'github' => array(
       'name' => 'GitHub',
-      'fa-icon' => 'fa-github-square',
+      'fa-icon' => 'fa-github',
+      'fa-icon-square' => 'fa-github-square',
     ),
     'google' => array(
       'name' => 'Google+',
-      'fa-icon' => 'fa-google-plus-square',
+      'fa-icon' => 'fa-google-plus',
+      'fa-icon-square' => 'fa-google-plus-square',
     ),
     'youtube' => array(
       'name' => 'YouTube',
-      'fa-icon' => 'fa-youtube-square',
+      'fa-icon' => 'fa-youtube',
     ),
     'flickr' => array(
       'name' => 'Flickr',
       'fa-icon' => 'fa-flickr',
     ),
     'linkedin' => array(
       'name' => 'LinkedIn',
-      'fa-icon' => 'fa-linkedin-square',
+      'fa-icon' => 'fa-linkedin',
```

Comment:
I think we might me missing a `fa-linkedin-square` here, http://fontawesome.io/icon/linkedin-square.